druid/src/test/java/org/apache/calcite/test/DruidAdapterIT.java

```diff
@@ -1695,28 +1694,120 @@ public Void apply(ResultSet resultSet) {
         + "where EXTRACT( year from \"timestamp\") = 1997 and "
         + "\"cases_per_pallet\" >= 8 and \"cases_per_pallet\" <= 10 and "
         + "\"units_per_case\" < 15 ";
-    String druidQuery = "{'queryType':'select','dataSource':'foodmart','descending':false,"
-        + "'intervals':['1900-01-09T00:00:00.000/2992-01-10T00:00:00.000'],"
-        + "'filter':{'type':'and',"
-        + "'fields':[{'type':'bound','dimension':'cases_per_pallet','lower':'8',"
-        + "'lowerStrict':false,'ordering':'numeric'},"
-        + "{'type':'bound','dimension':'cases_per_pallet','upper':'10','upperStrict':false,"
-        + "'ordering':'numeric'},{'type':'bound','dimension':'units_per_case','upper':'15',"
-        + "'upperStrict':true,'ordering':'numeric'}]},'dimensions':[],'metrics':['store_sales'],"
-        + "'granularity':'all','pagingSpec':{'threshold':16384,'fromNext':true},"
-        + "'context':{'druid.query.fetch':false}}";
+    String druidQuery = "{\"queryType\":\"timeseries\",\"dataSource\":\"foodmart\","
```

Comment:
Is it possible to use single quotes for DruidQuery instead of escaping the double quotes?